### DISCUSSION TITLE
Multistroke

### DISCUSSION THREAD

**Discussion by KoljaL** [sezanzeb/input-remapper#194] [Q&A] [answered]
# First of all, thank you for this great tool  ![](https://github.githubassets.com/images/icons/emoji/unicode/1f44d.png)
What I didn't find in the doc, the discussions or the issues is a way to interpret multiple keystrokes.

My concern is actually quite simple;
Since I never need the `CapsLock` key, I would like to have:

- 1 keystroke:   `Ctrl + c`
- 2 keystrokes: `Ctrl + v`
- hold down: `Ctrl + x`

Is there a way to do this?

**Comment by sezanzeb** [comment, marked as answer]
I need to fix a bug, then

```
if_tap(
  if_tap(
    m(Control_L, k(v)),
    m(Control_L, k(c))
  ),
  m(Control_L, k(x))
)
```

will work. I'll let you know when its ready

**Reply by sezanzeb** [reply]
Just pushed to main, I hope it works for you after installing from the latest source

**Reply by KoljaL** [reply]
Hey Tobi,
Thanks for the superfast implementation of my request.
I also have understood the principle of the macros. Previously, I had always tried to implement the functions (if_tab) on the key side.
Now I'm looking for more keys that I can assign multiple times ;-)
